framework/doc/content/documentation/systems/ICs/RandomIC.md

```diff
@@ -1,12 +1,15 @@
-<!-- MOOSE Documentation Stub: Remove this when content is added. -->
-
 # RandomIC
 
-!alert construction title=Undocumented Class
-The RandomIC has not been documented, if you would like to contribute to MOOSE by
-writing documentation, please see [/generate.md]. The content contained on this page explains
-the typical documentation associated with a MooseObject; however, what is contained is ultimately
-determined by what is necessary to make the documentation clear for users.
+RandomIC produces a field of random numbers in the range specified by its `min` and `max` parameters (inclusive).
```

Comment:
"min" and "max"

It seems that you are using " for parameter names, which I prefer. Single quotes should be for code.

Reply:
FYI, in the near future we I will add some sort of syntax for pulling parameter stuff from the JSON: "[p:RandomIC/max]". This will protect against spelling mistakes.